src/scripts/main.js

```diff
@@ -1,3 +1,55 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', function() {
+    resolve();
+  });
+
+  document.body.addEventListener('click', function() {
+    reject(new Error('error'));
+  });
+});
```

Comment:
There is no such requirement in task description.
Promise1 should only be resolved on logo click.
You can leave it, but then please provide some styles to your error message